### PR TITLE
AZP: add warning for azure

### PR DESCRIPTION
**Pull request by avildema**
Added warning message for azure pipeline

### CODE REVIEW

File: contrib/test_jenkins.sh [hunk 1]
```diff
@@ -92,6 +92,22 @@ then
...
+    test "x$RUNNING_IN_AZURE" = "xyes" && azp_tag=##vso[task.logissue type=warning]
+    echo "${azp_tag}${msg}"
+}
+
+# Report an error message to Azure pipeline
+log_error() {
+	msg=$1
+	azp_tag=""
+    test "x$RUNNING_IN_AZURE" = "xyes" && azp_tag=##vso[task.logissue type=error]
+    echo "${azp_tag}${msg}"
```
Comment (yosefe) [inline contrib/test_jenkins.sh:108]: fix indents

File: contrib/test_jenkins.sh [hunk 2]
```diff
@@ -92,6 +92,26 @@ then
...
+}
+
+# Report an error message to Azure pipeline
+log_error() {
+	msg=$1
+	azp_tag=""
+	set +x
+    test "x$RUNNING_IN_AZURE" = "xyes" && azp_tag=##vso[task.logissue type=error]
+    echo "${azp_tag}${msg}"
+    set -x
```
Comment (yosefe) [inline contrib/test_jenkins.sh:112]: indent

File: contrib/test_jenkins.sh [hunk 3]
```diff
@@ -92,6 +92,26 @@ then
...
 echo "==== Running on $(hostname), worker $worker / $nworkers ===="
 
+# Report an warning message to Azure pipeline
+log_warning() {
+	msg=$1
+	azp_tag=""
+	set +x
+    test "x$RUNNING_IN_AZURE" = "xyes" && azp_tag=##vso[task.logissue type=warning]
+    echo "${azp_tag}${msg}"
+    set -x
```
Comment (yosefe) [inline contrib/test_jenkins.sh:102]: indent

File: contrib/test_jenkins.sh [hunk 4]
```diff
@@ -92,6 +92,26 @@ then
 fi
 echo "==== Running on $(hostname), worker $worker / $nworkers ===="
 
+# Report an warning message to Azure pipeline
+log_warning() {
+	msg=$1
+	azp_tag=""
+	set +x
+    test "x$RUNNING_IN_AZURE" = "xyes" && azp_tag=##vso[task.logissue type=warning]
```
Comment (yosefe) [inline contrib/test_jenkins.sh:100]: add " " when setting azp_tag
https://dev.azure.com/ucfconsort/0b36e3f0-8ab9-4a48-b68b-4b2350e02c88/_apis/build/builds/11401/logs/135
```
2020-12-01T12:26:58.3216184Z ##[warning]'##vso[task.logissue==== Not building with latest gcc compiler ====' contains logging command keyword '##vso', but it's not a legal command. Please see the list of accepted commands: https://go.microsoft.com/fwlink/?LinkId=817296
```

**Review by yosefe — COMMENTED** [review]
can we just source az-helpers.sh in tests.yml, to reuse azure_log_warning and co?

**Comment by avildema** [comment]
> can we just source az-helpers.sh in tests.yml, to reuse azure_log_warning and co?

I think about it , better I can remove log function from az-helpers.sh,

**Comment by yosefe** [comment]
> I think about it , better I can remove log function from az-helpers.sh,

can't remove, we used them in other places, such as code style

**Comment by avildema** [comment]
bot:pipe:retest

**Review by yosefe — APPROVED** [review]

**Pull request merged**